Demos/Basic-ObjC/Basic-ObjC/AppDelegate/AppDelegate.m

```diff
@@ -26,9 +26,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     NSDictionary *dictionary = @{@"address" : @"Burger Park 667",
                                  @"end_date" : @"2017-10-31 23:00:00 +00:00",
+								 //@"end_time" : @"2017-10-31 23:00:00 +00:00",
                                  @"first_name" : @"Ola",
                                  @"last_name" : @"Nordman",
-                                 @"start_date" : @"2014-10-31 23:00:00 +00:00"
+                                 @"start_date" : @"2014-10-31 23:00:00 +00:00",
+								 @"start_time" : @"2014-10-31 23:00:00 +00:00",
```

Comment:
Is this still an issue since I removed the commented line above? It looks fine in my Xcode?

![screen shot 2015-04-16 at 14 59 15](https://cloud.githubusercontent.com/assets/677629/7182442/3b792dfa-e449-11e4-8e27-b3b0f4322617.png)